examples/llama-3/lora-1b-ray.yml

```
@@ -0,0 +1,79 @@
+base_model: NousResearch/Llama-3.2-1B
+# Automatically upload checkpoint and final model to HF
+# hub_model_id: username/custom_model_name
+
+load_in_8bit: false
```

We should set this to true for LORA
```suggestion
load_in_8bit: true
```

---

I think it's fine leaving it as a half precision lora for now, especially when using deepspeed too. Pretty sure deepspeed with 8bit lora's isn't well supported on some hardware (like h100s)